### DISCUSSION TITLE
How can I put a cancel button on Janssen's login screen?

### DISCUSSION THREAD

**Discussion by ghost** [JanssenProject/jans#1178] [Q&A] [answered]
How can I put a cancel button on Janssen's login screen?

**Comment by moabu** [comment]
Whats the purpose of the cancel button ? meaning what is the expected behavior after the user presses that button ?

**Comment by ghost** [comment, marked as answer]
Ignore it as the purpose is already being fulfilled in the newer version of Janssen by Allow/Deny button which appears automatically without enabling a consent script. 
The expected behaviour is to return back to redirect Url as a request is denied by the user.

**Reply by moabu** [reply]
Yep which is why I was asking. Happy to help
